tools/apprunner-gradle-plugin/src/main/java/org/projectnessie/quarkus/gradle/QuarkusApp.java

```diff
@@ -68,7 +68,7 @@ protected QuarkusApp(RunningQuarkusApplication runningApp) {
 
   public static AutoCloseable newApplication(Configuration configuration, Project project, Properties props) {
 
-    Configuration deploy = project.getConfigurations().create("quarkusAppDeploy");
+    Configuration deploy = project.getConfigurations().create("nessieQuarkusDeploy");
```

Comment:
nit: the original `quarkusAppDeploy` was there as this could theoretically be reusable and pushed to Quarkus in the future. Is the change signifying that this code is un-reusable?

Reply:
TBH, this configuration isn't actually be accessible, because you create it when the task executes. Not sure whether that's still possible in newer Gradle versions.